src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivity.test.js

```diff
@@ -50,6 +50,21 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivity',
 
     const getWrapper = (props = {}) => shallow(<AnnotationActivity {...mockActivity} {...props} />);
 
+    const defaultFeatures = {
+        activityFeed: {
+            modifyAnnotations: {
+                enabled: true,
+            },
+        },
+    };
+
+    const renderWithFeatures = (props, features) =>
```

Comment:
Maybe `getWrapperWithFeatures`? Also is `features` argument needed? Doesn't seem like it's used